### DISCUSSION TITLE
how to make mixed corner radius?

### DISCUSSION THREAD

**Discussion by theaddonn** [TomSchimansky/CustomTkinter#1914] [Q&A]
with mixed I mean a differnt corner radius for different corners? I tried a tuple like (0, 0, 10, 0) but it didn't work!...

**Comment by avalon60** [comment]
You can't. CustomTkinter does not work that way. The corner_radius is a natural number, not a vector. Setting the one  value, governs all corners.

**Comment by ElectricCandlelight** [comment]
Going by the (0,0,10,0) you want only one rounded corner and the rest at right angles?

You could use ` _background_corner_colors` which is a tuple and change 3 of the colours to match the background. Will give the same result.

**Reply by theaddonn** [reply]
oh ok!!

**Reply by avalon60** [reply]
Be aware that referencing leading underscore variables is a risk, and is considered bad practice. Variables are named as such, to indicate that they are for internal use only (in this case by CustomTkinter) . There is no guarantee that these won't change, disappear, or be subject to internal implementation changes in the future.

**Reply by newor0599** [reply]
@Adrian8115 can you show how will the code look like?
